kivy/weakmethod.py

```diff
@@ -1,4 +1,4 @@
-'''
+"""
```

Comment:
@matham Is Kivy standard to use single-quote strings instead of double-quote for docstring?

Reply:
It's not exactly a standard. That's just what some of the core-devs used. Personally, I used that too in my personal projects after seeing that in kivy. But pycharm complains about this, and looking at other projects double quotes are much more common.

That's why I consciously used double quotes here because I'd like to use that going forward. Although maybe that's not a good idea mixing different quoting styles.

Reply:
black is a tool that pushes double quotes as the default everywhere it can, and the pep 257 also advises to use them for docstrings https://www.python.org/dev/peps/pep-0257/#specification

Reply:
I would go with pep 257 too, but I think single-quote is dominant for docstrings in Kivy, so I would stick with single-quote until Kivy starts using black or someone does single to double change for all docstrings.